docs/Configuration.md

```diff
@@ -123,3 +123,23 @@ Optionally disable syncing to disk per transaction. Nosync being true means much
   * Default: "cayley"
 
 The name of the database within MongoDB to connect to. Manages its own collections and indicies therein.
+
+## Per-Replication Options
```

Comment:
Right notion. I even would have called it `replication_options` myself before. But I'm open to other suggestions, now that I see it in writing

(Thank you for keeping the docs in sync, btw!)